.github/workflows/ci_cd.yml

```diff
@@ -802,27 +613,28 @@ jobs:
           cname: docs.openverse.org
 
       - uses: peter-evans/find-comment@v2
+        if: github.event_name == 'pull_request'
         id: final-preview-comment
         with:
           issue-number: ${{ github.event.pull_request.number }}
           body-includes: Full-stack documentation
 
       - uses: peter-evans/create-or-update-comment@v2
+        if: github.event_name == 'pull_request'
         with:
+          issue-number: ${{ github.event.pull_request.number }}
           comment-id: ${{ steps.final-preview-comment.outputs.comment-id }}
           edit-mode: replace
           body: |
-            **Full-stack documentation**: _Ready_
-
-            <https://${{ env.REPO_OWNER }}.github.io/${{ env.REPO_NAME }}/_preview/${{ github.event.pull_request.number }}>
+            **Full-stack documentation**: <https://docs.openverse.org/_preview/${{ github.event.pull_request.number }}>
 
             Please note that GitHub pages takes a little time to deploy newly pushed code, if the links above don't work or you see old versions, wait 5 minutes and try again.
 
             You can check [the GitHub pages deployment action list](https://github.com/${{ github.repository }}/actions/workflows/pages/pages-build-deployment) to see the current status of the deployments.
 
       - name: Checkout repository # again, to enable cleaning
         uses: actions/checkout@v3
-        if: always()
+        if: success() || failure()
```

Comment:
Ah, nice detail here. I had to read about the nuance of `always` vs combining success and failure here for others who aren't aware of what the implication of this change is: https://docs.github.com/en/actions/learn-github-actions/expressions#always